circular reference when transpiling

hey guys,

found a bug while trying to transpile my project with the react-pdf/renderer as dependency. it triggers a circular reference on me. It works with version 2.0.6. But not with 2.0.8.

Typescript Version: 4.2.4
Renderer Version: 2.0.8

node_modules/@react-pdf/renderer/index.d.ts:19:10 - error TS2456: Type alias 'Style' circularly references itself.
19     type Style = Style;

this is the commit:
https://github.com/diegomura/react-pdf/commit/c72ff88e80ea07d8ccd05f1b40fde3b97d6b92dc

thanks for the tool and support.

--- a/packages/renderer/index.d.ts
+++ b/packages/renderer/index.d.ts
@@ -15,9 +15,6 @@ declare module '@react-pdf/renderer' {
     interface Styles {
       [key: string]: Style;
     }
-
-    type Style = Style;
-
     interface OnRenderProps {
       blob?: Blob;
     }

